scripts/load-test/collect-pod-logs/collect-openshift-pod-logs.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
+
+# Array of default namespaces
+default_namespaces=(
+    "application-service"
+    "build-service"
+    "enterprise-contract-service"
+    "gitops-service-argocd"
```

Comment:
@naftalysh Could you also add openshift-monitoring and openshift-user-workload-monitoring. Note here, openshift-user-workload-monitoring is not available by default. Needs to be enabled via this JIRA, STONEO11Y-86[](https://issues.redhat.com/browse/STONEO11Y-86).

Reply:
All the rules are defined in openshift-user-workload-monitoring. Will get back to you if o11y really needs openshift-monitoring or not.

Reply:
@srivickynesh 
As for the namespaces you asked me to add, the script is flexible and allows you also to add namespaces into the namespaces.txt file - I shall add them including openshift-user-workload-monitoring which is as you mentioned 
created as the user workload monitoring Prometheus is deployed as part of every CI run in Stonesoup.

We run scripts/load-test/collect-pod-logs/collect-openshift-pod-logs.sh script from machine logged into the CI cluster and not as part of the CI itself so I guess we shall see the openshift-user-workload-monitoring namespace as well.

Reply:
okay. Thanks